lm_eval/evaluator.py

```diff
@@ -55,17 +52,22 @@ def simple_evaluate(
         Dictionary of custom task descriptions of the form: `task_name: description`
     :param check_integrity: bool
         Whether to run the relevant part of the test suite for the tasks
+    :param seed: int
+        Random seed.
+    :param parallelize: bool
+        Whether to parallelize the model across gpus.
     :return
         Dictionary of results
     """
-    set_seed(1234)
+    set_seed(seed)
```

Comment:
Good catch!

Reply:
I think this was fixed once or twice before -- some merge issues have put it back /: